src/main/webapp/assets/modules/contests/challengeSearchForm.js

```diff
@@ -0,0 +1,18 @@
+techlooper.directive('challengeSearchForm', function () {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/contests/challengeSearchForm.html",
+    link: function (scope, el, attrs) {
```

Comment:
'attrs' is defined but never used.<br>'el' is defined but never used.<br>'scope' is defined but never used.